src/group.rs

```diff
@@ -163,3 +163,74 @@ impl Group for RistrettoPoint {
         constant_time_eq::constant_time_eq(&self.to_arr(), &other.to_arr())
     }
 }
+
+#[cfg(feature = "p256")]
+impl Group for p256::ProjectivePoint {
```

Comment:
I've used `ProjectivePoint`, but I don't know if that's actually correct.

Reply:
As far as I understand this doesn't really matter, but it makes necessary math more accessible.